package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "zn-service-util",
-  "version": "1.0.2",
-  "description": "zn-service-util",
```

Comment:
@JoeBello also make sure there is a lint script in package.json and that it's actually passing! :)